news/20200422.misc

```diff
@@ -0,0 +1 @@
+Add a version specifier to the Python dependancies for Mbed packages
```

Comment:
```suggestion
Add a version specifier to the Python dependencies for Mbed packages
```

Reply:
Damn sorry, missed this for some reason :(